### PR TITLE
BlastApps.FluentSearch version latest

### PR DESCRIPTION
**Pull request by hpr1999**
Official website: [https://www.fluentsearch.net](https://www.fluentsearch.net)
Github presence: [https://github.com/adirh3/Fluent-Search](https://github.com/adirh3/Fluent-Search)
Rights to this application belong to @adirh3 (https://github.com/adirh3)

- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/19275)

### CODE REVIEW

File: manifests/b/BlastApps/FluentSearch/latest/BlastApps.FluentSearch.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: BlastApps.FluentSearch
+PackageVersion: latest
```
Comment (ItzLevvie) [inline manifests/b/BlastApps/FluentSearch/latest/BlastApps.FluentSearch.installer.yaml:5]:
```suggestion
PackageVersion: 0.9.87.1
```

File: manifests/b/BlastApps/FluentSearch/latest/BlastApps.FluentSearch.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: BlastApps.FluentSearch
+PackageVersion: latest
```
Comment (ItzLevvie) [inline manifests/b/BlastApps/FluentSearch/latest/BlastApps.FluentSearch.yaml:5]:
```suggestion
PackageVersion: 0.9.87.1
```

File: manifests/b/BlastApps/FluentSearch/latest/BlastApps.FluentSearch.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: BlastApps.FluentSearch
+PackageVersion: latest
```
Comment (ItzLevvie) [inline manifests/b/BlastApps/FluentSearch/latest/BlastApps.FluentSearch.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 0.9.87.1
```

File: manifests/b/BlastApps/FluentSearch/latest/BlastApps.FluentSearch.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: BlastApps.FluentSearch
+PackageVersion: latest
+PackageLocale: en-US
+Publisher: BlastApps
+PackageName: Fluent Search
```
Comment (ItzLevvie) [inline manifests/b/BlastApps/FluentSearch/latest/BlastApps.FluentSearch.locale.en-US.yaml:8]:
```suggestion
PackageName: Fluent Search Beta
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Comment by ItzLevvie** [comment]
Could you please suppress this?
image

**Comment by hpr1999** [comment]
> Could you please suppress this?
> image

I'll try to have a look, thanks!

**Comment by hpr1999** [comment]
> > Could you please suppress this?
> > image
> 
> I'll try to have a look, thanks!

With which installation parameters does this appear? Upon installing, I simply get prompted for Admin permissions, both with and without the `-h` flag. Maybe that is because I have previously installed the software?

As a matter of curiosity, are Admin-prompts also considered bad taste for the installation?

File: manifests/b/BlastApps/FluentSearch/latest/BlastApps.FluentSearch.installer.yaml [hunk 5]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: BlastApps.FluentSearch
+PackageVersion: latest
+Installers:
+- Architecture: x64
+  InstallerType: inno
+  InstallerUrl: https://stfluentsearch.blob.core.windows.net/fluent-search/Fluent-Search.exe
+  InstallerSha256: 66AD29D6F7D3847EB7852B1FC0FB8E0AC4F9F37AAC9B263B3A4FC395A7A8F412
```
Comment (ItzLevvie) [inline manifests/b/BlastApps/FluentSearch/latest/BlastApps.FluentSearch.installer.yaml:10]:
```suggestion
Installers:
- Architecture: x64
  InstallerType: inno
  InstallerUrl: https://stfluentsearch.blob.core.windows.net/fluent-search/Fluent-Search.exe
  InstallerSha256: 66AD29D6F7D3847EB7852B1FC0FB8E0AC4F9F37AAC9B263B3A4FC395A7A8F412
  Scope: user
  InstallerSwitches:
    Custom: /SUPPRESSMSGBOXES /CURRENTUSER
- Architecture: x64
  InstallerType: inno
  InstallerUrl: https://stfluentsearch.blob.core.windows.net/fluent-search/Fluent-Search.exe
  InstallerSha256: 66AD29D6F7D3847EB7852B1FC0FB8E0AC4F9F37AAC9B263B3A4FC395A7A8F412
  Scope: machine
  InstallerSwitches:
    Custom: /SUPPRESSMSGBOXES /ALLUSERS
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/b/BlastApps/FluentSearch/latest/BlastApps.FluentSearch.locale.en-US.yaml (hunk 4 above)
Comment (hpr1999) [reply manifests/b/BlastApps/FluentSearch/latest/BlastApps.FluentSearch.locale.en-US.yaml:8]: I'm not sure how appropriate this is. It's still in the description on the Windows store and the Version Number is <1, but I think it does not feel like beta software. Maybe @adirh3 could weigh in, but it's probably not a big deal either way.

File: manifests/b/BlastApps/FluentSearch/latest/BlastApps.FluentSearch.installer.yaml (hunk 5 above)
Comment (hpr1999) [reply manifests/b/BlastApps/FluentSearch/latest/BlastApps.FluentSearch.installer.yaml:10]: Thanks for your work, that looks good!

File: manifests/b/BlastApps/FluentSearch/latest/BlastApps.FluentSearch.locale.en-US.yaml (hunk 4 above)
Comment (ItzLevvie) [reply manifests/b/BlastApps/FluentSearch/latest/BlastApps.FluentSearch.locale.en-US.yaml:8]: According to the latest docs over at https://github.com/microsoft/winget-pkgs/blob/master/doc/manifest/schema/1.0.0/defaultLocale.md#packagename, the name has to match with what's shown in Control Panel.

If it's out of beta, you'll have to contact the application's developer to update the name.

Comment (hpr1999) [reply manifests/b/BlastApps/FluentSearch/latest/BlastApps.FluentSearch.locale.en-US.yaml:8]: Oh, alright. He's contacted me privately, the Beta name is still correct, sorry for any inconveniences on your end, you've done most of the work here.

**Comment by hpr1999** [comment]
Is there a way for me to apply the suggestions you made? It says those need to be made by a user with write-access.
Or should I manually change them, commit and push the new version to the branch?
(Please excuse my ignorance of the process here)

**Comment by hpr1999** [comment]
I have accidentally deleted the forked repo and therefore, this is unnecessarily complicated - sorry for that. However, the creator of the app has opened a new pull request with the files including @ItzLevvie 's suggestions at https://github.com/microsoft/winget-pkgs/pull/19337

**Comment by OfficialEsco** [comment]
There should be a button next to the suggestion which says Commit Changes

**Comment by hpr1999** [comment]
@OfficialEsco I've looked there, because I know that feature from Gitlab. I couldn't find it, perhaps that was because I stupidly deleted the forked repo. Thanks for following up, anyway!

**Pull request closed without merging**